hyperpage.cpp

```diff
@@ -133,22 +133,31 @@ hyperpage::writer::writer(const std::string &db_path) : _handle(nullptr, close_h
     {
         throw std::runtime_error("Failed to open database: " + db_path);
     }
+    
+    // Set journal mode to DELETE to avoid WAL mode locking issues
+    sqlite3_exec(db, "PRAGMA journal_mode=DELETE;", nullptr, nullptr, nullptr);
+    
+    // Create table
     const std::string create_table_query =
         "CREATE TABLE IF NOT EXISTS hyperpage ("
         "path TEXT PRIMARY KEY, "
         "mime_type TEXT, "
-        "content BLOB);"
-        "CREATE UNIQUE INDEX IF NOT EXISTS path_index ON hyperpage (path);";
+        "content BLOB);";
```

Comment:
This query already worked, I am not convinced that it is actually a problem in SQLite to submit a multiline query.